scipy/stats/stats.py

```diff
@@ -3386,8 +3388,14 @@ def ttest_ind(a, b, axis=0, equal_var=True):
         If False, perform Welch's t-test, which does not assume equal
         population variance [2]_.
         .. versionadded:: 0.11.0
-
-
+    permutations : int, optional
+        The number of permutations that will be used to calculate p-values
+        using a permutation test.  The permutation test will only be run
+        if permutations > 0.
+        .. versionadded:: 0.15.2
```

Comment:
I fixed this and a number of other small things already in my branch linked in the comments.